packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -231,6 +231,12 @@ export class Pipeline extends PipelineBase {
         encryption: s3.BucketEncryption.KMS,
         removalPolicy: RemovalPolicy.RETAIN
       });
+      // add an alias to make finding the key in the console easier
+      new kms.Alias(this, 'ArtifactsBucketEncryptionKeyAlias', {
+        aliasName: this.generateNameForDefaultBucketKeyAlias(),
+        targetKey: encryptionKey,
+        removalPolicy: RemovalPolicy.RETAIN, // alias should be retained, like the key
```

Comment:
Does that not mean that destroying and redeploying the stack will fail?

Reply:
Yes, but that was always true, even before adding the `Alias` (because of the orphaned, physically-named S3 Bucket backing the Pipeline).

Reply:
Pinging about this.